packages/cfd/src/Containers/cfd-reset-password-modal.tsx

```diff
@@ -127,7 +127,12 @@ const CFDResetPasswordModal = ({
         };
 
         WS.tradingPlatformInvestorPasswordReset(request).then((response: { error: TError; password_type: string }) => {
-            if (response.error && (response.error.code === 'InvalidToken' || response.error.code === 'BadSession')) {
+            if (
+                response.error &&
+                (response.error.code === 'InvalidToken' ||
+                    response.error.code === 'BadSession' ||
+                    response.error.code === 'SameAsMainPassword')
```

Comment:
is not it possible to get any other error code rather than these three(InvalidToken, BadSession, SameAsMainPassword ) for the tradingPlatformInvestorPasswordReset request?

Reply:
We just have these three yet but I can have an else if condition just in case new errors are made from the BE.
The chance of having new error codes are so low but having the else if is more assuring to have a better code.

Reply:
Yes, it's always possible (PermissionDenied and InternalServerError are the most typical). The front-end will show `success` message for any error code out of this list, right? It looks odd.

Reply:
Do we have a message sent from the BE while we get PermissionDenied and InternalServerError ?
Having a modal with no message shown is a bit weird in terms of UX.

Reply:
Yes. Error code always come with an error message.